src/app/(pages)/groups/page.tsx

```diff
@@ -3,31 +3,37 @@ import Paginator from "@/components/paginator";
 import { getGroupsPage } from "@/services/groups";
 import { InputQuery } from "@/components/inputs";
 import { AddGroupButton, GroupsTable } from "./components";
+import { Suspense } from "react";
 
 type GroupsProps = {
   searchParams?: Promise<{
     count?: string;
     page?: string;
     query?: string;
+    me?: string;
   }>;
 };
 
 export default async function GroupsPage(props: Readonly<GroupsProps>) {
   const searchParams = await props.searchParams;
+  const isMe = searchParams?.hasOwnProperty("me");
   const count = searchParams?.count ? parseInt(searchParams.count) : 10;
   const page = searchParams?.page ? parseInt(searchParams.page) : 1;
   const query = searchParams?.query;
   const startIndex = 1 + count * (page - 1);
-  const groupsPage = await getGroupsPage(count, startIndex, query);
-  const numberOfPages = Math.ceil(groupsPage.totalResults / count);
+  const groupsPage = await getGroupsPage(count, startIndex, isMe, query);
```

Comment:
This is ok since it is consistent with the `users` clients page.
Actually I would prefer using the parading of the `client` page, splitting the function in two, such as
`getMyGroupsPage()` and `getGroupsPage()` and call the like `const groups = isMe ? getMyGroupsPage() : getGroupsPage()`